src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -125,7 +141,17 @@ public VerificationResult verifyPhoneWithCode(Request req, Response res) {
                 "Unknown error encountered while checking SMS verification code"
             );
         }
-        // If the check is successful, status will be "approved"
-        return new VerificationResult(check);
+        // If the check is successful, status will be "approved".
+        VerificationResult verificationResult = new VerificationResult(check);
+        if (verificationResult.status.equals("approved")) {
+            // If the check is successful, update the OtpUser
+            // (set isPhoneNumberVerified and notificationChannel).
+            // TODO: Figure out adding a test for this update to the OtpUser record.
+            otpUser.isPhoneNumberVerified = true;
+            otpUser.notificationChannel = "sms";
```

Comment:
I don't think we should be setting channel here. I think that should be independent of this action (especially considering that we intend to allow a "both" option for notification channel).

Reply:
Removed. I was on the fence about that.